src/python/zquantum/qcbm/cost_function.py

```diff
@@ -7,73 +8,37 @@
 )
 from zquantum.core.circuit import build_ansatz_circuit
```

Comment:
I know it wasn't introduced in this PR but this import is unused so we might as well remove it now :slightly_smiling_face:

Reply:
Addressed here: 
5892e6c